src/libcollections/vec.rs

```diff
@@ -1505,6 +1505,22 @@ impl<'a> From<&'a str> for Vec<u8> {
 // Clone-on-write
 ////////////////////////////////////////////////////////////////////////////////
 
+#[stable(feature = "rust1", since = "1.7.0")]
```

Comment:
This'll need a different feature name because the `rust1` feature was stabilized in 1.0.0